springwolf-examples/springwolf-cloud-stream-rabbit-binder-example/build.gradle

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id 'java'
+    id 'org.springframework.boot' version '2.6.2'
+    id 'io.spring.dependency-management' version '1.0.6.RELEASE'
+    id 'com.bmuschko.docker-spring-boot-application' version '6.1.4'
+    id 'org.jetbrains.kotlin.jvm' version '1.7.0'
+}
+
+sourceCompatibility = '1.8'
+version '0.3.1'
+
+repositories {
+    // For local development with snapshots
+    mavenLocal() {
+        content {
+            includeGroup "io.github.springwolf"
+        }
+    }
+
+    mavenCentral()
+}
+
+dependencies {
+    implementation project(":springwolf-plugins:springwolf-spring-cloud-rabbit-binder-plugin")
+    runtimeOnly 'io.github.springwolf:springwolf-ui:0.4.0'
+
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.cloud:spring-cloud-stream-binder-rabbit:3.2.4'
```

Comment:
I think that we should avoid specifying a version for the spring dependencies

Reply:
I HATED putting a version number here.  but how do I get around this?
![image](https://user-images.githubusercontent.com/13246477/177429078-aa2e984d-e602-4123-9a3c-4a6d20adabed.png)